site/content/blogs/release-v131.en.md

```diff
@@ -18,9 +18,11 @@ Thanks to every one of you who shows love and support for AWS Copilot.
 
 Copilot v1.31 brings big enhancements to help you develop more flexibly and efficiently:
 
-- **NLB enhancements**: You can now add security groups to Copilot-managed NLBs. NLBs also support the UDP protocol.
+- **NLB enhancements**: You can now add security groups to Copilot-managed [network load balancers](../docs/manifest/lb-web-service.en.md#nlb). NLBs also support the UDP protocol.
 - **Better task failure logs**: Copilot will show more descriptive information during deployments when tasks fail, allowing better troubleshooting.
-- **`copilot deploy` enhancements: You can now deploy multiple workloads at once, or deploy all local workloads with `--all`.
+- **`copilot deploy` enhancements**: You can now deploy multiple workloads at once, or deploy all local workloads with `--all`.
+
+#### Please read if you use `nlb`: [NLB security group enhacement](#nlb-enhancements)
```

Comment:
Maybe just more details at the end like https://github.com/aws/copilot-cli/commit/983ffd443ef2c817962ccba714cf27f459e1e0fb#diff-c933228c363a1118c64c52bd0c84346dbac20d74a73c370c3e0df06394003a10R24 ?

Reply:
I think we meant to ask users to take a look at the `!!!info` block, which is too much information to cram into one line : (

Reply:
hmm i meant adding "[See detailed section](#nlb-enhancements)" to the bullet point? It's just a shortcut and they'll read the section and notice `!!!info` anyway right 🤔

Reply:
Yeah, I don't know a way to link to a section that's not a header. So linking to nlb-enhancements should be fine. Do you think "more details" will give people enough of a heads up to read the section?

Reply:
yeah i think it's good enough. it's a relatively short blog post. I am also ok if we keep it but at least fix the typo